include/bluetooth/mesh/access.h

```diff
@@ -166,8 +166,13 @@ struct bt_mesh_model_op {
 	/** OpCode encoded using the BT_MESH_MODEL_OP_* macros */
 	const uint32_t  opcode;
 
-	/** Minimum required message length */
-	const size_t min_len;
+	/** Message length. If the message has variable length then this value
+	 *  indicates minimum message length and should be positive. Handler
+	 *  function should verify precise length based on the contents of the
+	 *  message. If the message has fixed length then this value should
+	 *  be negative.
+	 */
+	const ssize_t len;
```

Comment:
Can `opcode` and `len` be placed in a four byte address space. `opcode` occupy two space with model cid field, `len` occupy another two space.

Reply:
Does it not occupy 3 bytes? CID + number? Or is this assuming that the CID check PR goes in, and it can be ignored in this comparison?